frontend/src/views/MigrationHistoryDetail.vue

```diff
@@ -174,18 +205,23 @@ import { CodeDiff } from "v-code-diff";
 import MigrationHistoryStatusIcon from "../components/MigrationHistoryStatusIcon.vue";
 import { idFromSlug, nanosecondsToString } from "../utils";
 import {
+  Database,
   MigrationHistory,
   MigrationHistoryPayload,
   VCSPushEvent,
 } from "../types";
+import { BBSelect } from "../bbkit";
 
 interface LocalState {
   showDiff: boolean;
+  selectedOld: string;
+  oldSchema: string;
```

Comment:
Add a comment what's oldSchema and what's newSchema.

Also can we use a boolean or enum to represent selectedOld?